packages/gafl-webapp-service/src/pages/concessions/date-of-birth/date-of-birth.njk

```diff
@@ -40,11 +41,9 @@
 %}
 
 {% block pageContent %}
-    <fieldset class="govuk-fieldset" role="group" aria-describedby="passport-issued-hint">
-      <div id="passport-issued-hint" class="govuk-hint">
-        {{ mssgs.dob_entry_hint }}
-      </div>
-    </fieldset>
+    <div id="date-of-birth-hint" class="govuk-hint">
```

Comment:
So here, it should be `<div id="{{ describedByTag }}" class="govuk-hint">`

Reply:
Ahh good shout, will fix